README.md

```diff
@@ -3,42 +3,36 @@ SublimeLinter-gotype
+
 
 ## Installation
-SublimeLinter 3 must be installed in order to use this plugin. If SublimeLinter 3 is not installed, please follow the instructions [here][installation].
 
-### Linter installation
-Before using this plugin, you must ensure that `gotype` is installed on your system. To install `gotype`, do the following:
+SublimeLinter must be installed in order to use this plugin. 
+
+Please install via [Package Control](https://packagecontrol.io).
+
+Before using this plugin, ensure that `gotype` is installed on your system.
+To install `gotype`, do the following:
 
 1. From the command line type the following:
 
    ```
    go get -u golang.org/x/tools/cmd/gotype
    ```
 
-### Linter configuration
-In order for `gotype` to be executed by SublimeLinter, you must ensure that its path (`$GOPATH/bin`) is available to SublimeLinter. Before going any further, please read and follow the steps in [“Finding a linter executable”](http://sublimelinter.readthedocs.org/en/latest/troubleshooting.html#finding-a-linter-executable) through “Validating your PATH” in the documentation.
-
-Once you have installed and configured `gotype`, you can proceed to install the SublimeLinter-gotype plugin if it is not yet installed.
-
-### Plugin installation
-Please use [Package Control][pc] to install the linter plugin. This will ensure that the plugin will be updated when new versions are available. If you want to install from source so you can modify the source code, you probably know what you are doing so we won’t cover that here.
-
-To install via Package Control, do the following:
-
-1. Within Sublime Text, bring up the [Command Palette][cmd] and type `install`. Among the commands you should see `Package Control: Install Package`. If that command is not highlighted, use the keyboard or mouse to select it. There will be a pause of a few seconds while Package Control fetches the list of available plugins.
-
-1. When the plugin list appears, type `gotype`. Among the entries you should see `SublimeLinter-gotype`. If that entry is not highlighted, use the keyboard or mouse to select it.
 
 ## Settings
-For general information on how SublimeLinter works with settings, please see [Settings][settings]. For information on generic linter settings, please see [Linter Settings][linter-settings].
 
-In addition to the standard SublimeLinter settings, SublimeLinter-gotype provides its own settings. Those marked as “Inline Setting” or “Inline Override” may also be [used inline][inline-settings].
+- SublimeLinter settings: http://sublimelinter.com/en/latest/settings.html
+- Linter settings: http://sublimelinter.com/en/latest/linter_settings.html
 
-|Setting|Description         |Inline Setting|Inline Override|
-|:------|:-------------------|:------------:|:-------------:|
-|gopath |Set a custom GOPATH |||
+Additional SublimeLinter-gotype provides settings:
+
+|Setting|Description         |
+|:------|:-------------------|
+|gopath |Set a custom GOPATH |
```

Comment:
Is this still available? It looks like it's removed with your PR.

Reply:
Indeed, it’s gone. All linters take an “env” setting where you can set things like this.